doc/manuals/admin/cli.md

```diff
@@ -111,6 +111,8 @@ The list of available options is the following:
     broker process.
 -   **-httpTimeout <interval>**. Specifies the timeout in milliseconds
     for forwarding messages and for notifications.
+-   **-connTimeout <interval>**. Specifies the timeout in seconds
```

Comment:
Apart from this description in cli.md, this parameter probably deserves a new entry in https://fiware-orion.readthedocs.io/en/master/admin/perf_tuning/index.html#http-server-tuning.